### DISCUSSION TITLE
I can't find my University listed

### DISCUSSION THREAD

**Discussion by vreoo** [community/community#75858] [GitHub Education]
### Hi everyone,

Hi everyone,

I have already submitted a Benifets Application on Nov 9, 2021. The offer expired today, and I am trying to submit another one since I am still enrolled as a student at my University.

I've been struggling to find my University (`Arab Open University - Kuwait Branch` Website: https://www.aou.edu.kw/) since It's not getting recognized by Github for some reason at all, It shows other branches from other countries, but the branch in my country `Kuwait` is not recognized or found at all. The fact that our branch is considered the HQ of the other branches makes me laugh at how silly the situation is xD.

I am not sure what should I do from here, or who I should contact in this regard, So I thought I might drop this discussion ticket here and hopefully, I find some assistance.

Thanks in advance,

**Comment by amitpatle** [comment]
try to connect your university email with  github

**Reply by vreoo** [reply]
I have done this long time ago, because i had a subscription before but i am renewing it again now, but i couldnt proceed since i couldnt find my University listed.

**Reply by Aikpo** [reply]
Same here

**Reply by amitpatle** [reply]
Then there may some issue with your account: have you tried alt account if yes then GitHub unlisted your university due some reason.
